packages/broadway.rb

```diff
@@ -13,65 +13,25 @@ class Broadway < Package
   })
 
   depends_on 'gtk3'
-
-  def self.build
-    system "echo 'GDK_BACKEND=broadway' > .broadway.env"
-    system "echo 'XDG_RUNTIME_DIR=/var/run/chrome' >> .broadway.env"
-    system "echo 'BROADWAY_DISPLAY=:5' >> .broadway.env"
-    system "echo '#!/bin/bash' > initbroadway"
-    system "echo 'BROADWAYD=\$(pidof broadwayd 2>/dev/null)' >> initbroadway"
-    system "echo 'if [ -z \"\${BROADWAYD}\" ]; then' >> initbroadway"
-    system "echo '  [ -f #{CREW_PREFIX}/bin/stopsommelier ] && stopsommelier' >> initbroadway"
-    system "echo '  broadwayd \${BROADWAY_DISPLAY} &' >> initbroadway"
-    system "echo '  sleep 3' >> initbroadway"
-    system "echo 'fi' >> initbroadway"
-    system "echo 'BROADWAYD=\$(pidof broadwayd 2>/dev/null)' >> initbroadway"
-    system "echo 'if [ ! -z \"\${BROADWAYD}\" ]; then' >> initbroadway"
-    system "echo '  echo \"broadwayd process \${BROADWAYD} is running\"' >> initbroadway"
-    system "echo 'else' >> initbroadway"
-    system "echo '  echo \"broadwayd failed to start\"' >> initbroadway"
-    system "echo '  exit 1' >> initbroadway"
-    system "echo 'fi' >> initbroadway"
-    system "echo '#!/bin/bash' > stopbroadway"
-    system "echo >> stopbroadway"
-    system "echo 'BROADWAYD=\$(pidof broadwayd 2>/dev/null)' >> stopbroadway"
-    system "echo 'if [ ! -z \"\${BROADWAYD}\" ]; then' >> stopbroadway"
-    system "echo '  pkill broadwayd' >> stopbroadway"
-    system "echo '  sleep 3' >> stopbroadway"
-    system "echo 'fi' >> stopbroadway"
-    system "echo 'BROADWAYD=\$(pidof broadwayd 2> /dev/null)' >> stopbroadway"
-    system "echo 'if [ -z \"\${BROADWAYD}\" ]; then' >> stopbroadway"
-    system "echo '  echo \"broadwayd process stopped\"' >> stopbroadway"
-    system "echo 'else' >> stopbroadway"
-    system "echo '  echo \"broadwayd process \${BROADWAYD} is running\"' >> stopbroadway"
-    system "echo '  exit 1' >> stopbroadway"
-    system "echo 'fi' >> stopbroadway"
-  end
-
-  def self.install
-    system "install -Dm755 initbroadway #{CREW_DEST_PREFIX}/bin/initbroadway"
-    system "install -Dm755 stopbroadway #{CREW_DEST_PREFIX}/bin/stopbroadway"
-    system "install -Dm644 .broadway.env #{CREW_DEST_HOME}/.broadway.env"
-  end
-
+  
   def self.postinstall
+    puts 'To configure the Broadway daemon, remove the former'.lightblue
+    puts ' "Broadway environment variables + daemon" section'.lightblue
+    puts ' from `~/.bashrc`, and run this:'.lightblue
```

Comment:
And how exactly does this make it any easier for the end user?  You just made the transition more difficult.  This is what the bash scripts you conveniently removed already do automagically.

Reply:
Because the bash scripts screw stuff up. They're not part of the package, and they just make things harder.

Reply:
Can you explain exactly what you mean by screwing stuff up?  That is a generalization, not a specific issue that we can address.

Reply:
They set variables that cannot be unset, take ~15 seconds every shell startup, and display annoying debug messages.